build.sbt

```diff
@@ -144,30 +161,30 @@ lazy val root = (project in file(".")).aggregate(core, python, docs)
 lazy val scalaLoggingDependency = settingKey[ModuleID]("scalaLoggingDependency")
 ThisBuild / scalaLoggingDependency := {
   (ThisBuild / scalaVersion).value match {
-    case `scala211` => "com.typesafe.scala-logging" %% "scala-logging-slf4j" % "2.1.2"
-    case `scala212` => "com.typesafe.scala-logging" %% "scala-logging" % "3.7.2"
+    case `scalaOld` => "com.typesafe.scala-logging" %% "scala-logging-slf4j" % "2.1.2"
+    case `scalaNew` => "com.typesafe.scala-logging" %% "scala-logging" % "3.7.2"
     case _ =>
       throw new IllegalArgumentException(
-        "Only supported Scala versions are: " + Seq(scala211, scala212))
+        "Only supported Scala versions are: " + Seq(scalaOld, scalaNew))
   }
 }
 
 lazy val core = (project in file("core"))
   .settings(
     commonSettings,
     functionGenerationSettings,
-    name := "glow",
+    name := s"glow_spark${majorVersion(sparkVersion.value)}_scala",
```

Comment:
are you sure we want the `_scala`? sbt usually adds that automatically. Also, I think it's a bit more common to use `-` instead of `_` in artifact names. See search results for example: https://mvnrepository.com/search?q=spark2.

Reply:
changed _ to -. 
I think it is better to keep scala. sbt adds _2.11 or _2.12 which will look confusing: glow spark3_2.11. Even the currently release artifacts are confusing glow_2.11. It looks as if 2.11 is version of glow or part of version of spark.